src/power.rs

```diff
@@ -73,13 +71,11 @@ pub fn read_battery_charge() -> Result<i8, Error> {
     let battery_charge_path: Vec<&str> = vec![
         "/sys/class/power_supply/BAT/capacity",
         "/sys/class/power_supply/BAT0/capacity",
-        "/sys/class/power_supply/BAT1/capacity"
+        "/sys/class/power_supply/BAT1/capacity",
```

Comment:
maybe add a BAT2 as well just in case? ( i do have a BAT2 in my system but its empty so idk what happened there)

Reply:
yea